_ft/api.yml

```diff
@@ -159,11 +159,6 @@ paths:
       responses:
         200:
           description: Returns a canonicalized array of annotations that have been successufully saved in PAC.
```

Comment:
The descriptions for all responses should be updated.

Reply:
Yep, I forgot to do this.